lib/dialects/postgres/query-generator.js

```diff
@@ -281,7 +281,7 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
     return `ALTER TABLE ${quotedTableName} DROP COLUMN ${quotedAttributeName};`;
   }
 
-  changeColumnQuery(tableName, attributes) {
+  changeColumnQuery(tableName, attributes, dataTypeOrOptions) {
```

Comment:
this method has no reason to accept a dataType, this should just be `options`

```suggestion
  changeColumnQuery(tableName, attributes, options) {
```